analyzers/src/SonarAnalyzer.Vsix/SonarAnalyzer.Vsix.csproj

```diff
@@ -19,7 +18,7 @@
     <OutputType>Library</OutputType>
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>SonarAnalyzer.Vsix</RootNamespace>
-    <AssemblyName>SonarAnalyzer</AssemblyName>
+    <AssemblyName>SonarAnalyzer.Vsix</AssemblyName>
```

Comment:
It was the same assembly name with `SonarAnalyzer.Common`. NuGet restore can get confused due to this